### PR TITLE
Add Jupyter Notebook sample for helping ML model and dataset search

### LINKED ISSUE #46: Tool to tag and search Azure ML Datasets

### PR DESCRIPTION
**Pull request by angie4u**
This Jupyter Notebook helps users to search Azure ML model and dataset using tag information.
Please check the overall structure looks okay and the readme explains well the customer scenario.

### CODE REVIEW

**Comment by angie4u** [comment]
When you review this PR, it will be easier to review the jupyter notebook with file viewer like this! FYI! https://github.com/liupeirong/MLOpsManufacturing/blob/50d4b9773a61554f011d3fe080287a4e15b0bf84/common/dataset_search_tool/dataset_search_tool.ipynb

**Comment by h2floh** [comment]
Hey @angie4u , 

could you please also add a link to the Common Overview [README](https://github.com/liupeirong/MLOpsManufacturing/blob/eunk/dataset-search/common/README.md)

And would be great if you could describe what the notebook does in more detail. Since Jupyter Notebooks are just for that you can have a look [here](https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/training/train-on-amlcompute/train-on-amlcompute.ipynb) how it should ideally look like. You can also copy the Copyright and License hint just as is.

Last thing, I think we should rename the labels mentioned in the query. And currently in this repo we don't have a labeled dataset. Would it be possible to provide a sample dataset/description or script to create a labeled dataset and then modify the queries that they would show some result if you have that sample dataset in place?

File: common/dataset_search_tool/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## Business Scenario
+
+A lot of ML models and datasets are uploaded and managed using [Azure Machine Learning Service](https://azure.microsoft.com/en-us/services/machine-learning/). It can be effectively managed by adding meta information using [tags](https://docs.microsoft.com/en-us/python/api/azureml-core/azureml.data.abstract_dataset.abstractdataset?view=azure-ml-py#add-tags-tags-none-) in ml model and dataset.
+Such as tracking of used datasets within the automated processing of data, model creation and deployment (intermediate) datasets need to be tagged with metainformation about data quality, content, run in which they were generated and other data to identify or exclude them as candidates for a training run or for a easier review if quality of the model is not as expected.
+However, Azure Machine Learning Service portal currently does not provide a ui that can check a specific dataset or ml model using tag information. There was a request from a customer who wanted to search dataset or ML model using tag information conveniently. This jupyter notebook was created to support this kinds of request.
```
Comment (h2floh) [inline common/dataset_search_tool/README.md]:
```suggestion
However, Azure Machine Learning Service portal currently does not provide a UI that can check a specific dataset or ml model using tag information. This jupyter notebook was created to support this task.
```

**Review by h2floh — DISMISSED** [review]
Thanks @angie4u for this nice tool! Can you please check if you are ok with wording suggesting for the README?

**Comment by angie4u** [comment]
Hi @h2floh ! I updated the readme but I need your approval again for the latest change! 🤩

**Review by h2floh — APPROVED** [review]

**Pull request merged**